src/Lime.Protocol/Network/SenderChannel.cs

```diff
@@ -232,6 +232,10 @@ private async Task SendToTransportAsync()
                     {
                         break;
                     }
+                    catch (EnvelopeTooLargeException ex)
```

Comment:
Suggestion: Since this is a unique behavior, put it as the first "catch" clause and add a comment explaining why we do not throw the exception in this case.

Reply:
nice one, doing!

Reply:
did it